generators/app/templates/infrastructure/src/index.js

```diff
@@ -159,6 +160,8 @@ async function startServer(httpServer) {
               ctx.tenant = {};
             }
         <%_}_%>
+
+        ctx.state = {'jwtdata':decoded, 'token':token}
```

Comment:
why not:
```
ctx.state={ jwtdata: decoded, token }
```
no need for the extra " ' " and the value token has the same name as the desired property..